doc/contributing/maintaining-http.md

```diff
@@ -0,0 +1,98 @@
+# Maintaining HTTP
+
+Support for HTTP is a key priority in terms of ensuring the continued success of
+Node.js as captured in the projects
+[technical priorities](https://github.com/nodejs/node/blob/master/doc/contributing/technical-priorities.md).
+
+The current high level stragegy is based on the discussion in the
+[Next-10](https://github.com/nodejs/next-10)
+ [mini-summit](https://github.com/nodejs/next-10/blob/main/meetings/summit-jan-2022.md)
+on modern HTTP which was held on Jan 27 2022.
+
+## High level strategy
+
+The key elements of our strategy for future HTTP APIs are:
+
+* APIs should HTTP protocol independant (support HTTP1, HTTP2, etc.) 
+* APIs should be transport protocol independant (tcp, quick, etc.)
+* APIs should provide good defaults the perform well
+* Client/server APIs should be consistent and allow easy integration
+* Common requirements like piping out from client API to server APIs should be easy
+* For both the Client and Sever there is a need for multiple APIs, with each targeting
+  a different level of abstraction.
+
+Unfortunately our existing HTTP APIs:
+
+* [HTTP](https://nodejs.org/docs/latest/api/http.html)
+* [HTTPS](https://nodejs.org/docs/latest/api/https.html)
+* [HTTP2](https://nodejs.org/docs/latest/api/http2.html) 
+
+do not align with the key elements. For this reason, while these APIs
+are widely used and we do not plan to deprecate or remove them,
+they are not the focus of active development or performance improvements.
+Bug fixing is still important for all of these APIs.
+
+With respect to the HTTP protocols themselves our current assessment in
+terms of priority within existing or new APIs is:
+* HTTP 2 in “maintenance mode” both protocol and APIs
+* HTTP 1 stable protocol, but innovation in APIs
+* HTTP 3 is the future need to plan in support
+
+The current strategy is to build out 2 new client APIs and 2 new Server APIs
+in line with the key elements above.
```

Comment:
I'm still confused about this reference to "key elements", isn't this just referring to the "high level strategy" section? Maybe we should consistently refer to it as such?

Reply:
```suggestion
in line with the high level strategy above.
```